org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -29,6 +24,7 @@
             <unit id="org.eclipse.jdt.source.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
             <unit id="org.mockito.mockito-core" version="0.0.0"/>
+            <unit id="org.apache.commons.commons-io" version="0.0.0"/>
```

Comment:
Hold on. I think we can actually get rid of commons.io. I can't find it being used in jdt.ls.core, and @akurtakov may have removed it with 9e35f365f2fab5608127c06a36b910e616a31a93 . It's currently used in jdt.ls.tests in a few places but only for `org.apache.commons.io.FileUtils`, which Alex has shown how to remove in his commit.

If it's still used by platform, that's fine, but if you could just update jdt.ls.tests I think it should be possible to remove explicit mention of it.

Reply:
Right, nevemind. I see it's more than just FileUtils, it's some other methods :\